index.html

```diff
@@ -89,7 +89,7 @@ <h5> Exotics </h5>
         <img src="https://mershark.github.io/images/icons8-lecture-64.png" />
         <h5 >Electric cars</h5>
```

Comment:
- [ ] Kindly make sure to use a white background image for the Header section or add a white backdrop on top of the background image so your website matches the provided design

![Screenshot_20230706_201224](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/e963eb99-e6a4-4551-b6ab-bdb7a3538cc5)